### DISCUSSION TITLE
Changing Settings possible?

### DISCUSSION THREAD

**Discussion by stefka141** [phiresky/ebv-theta-to-mqtt#3] [Q&A] [answered]
Nice work 😀 
Is it also possible to change settings?

Not everything, but maybe the general ones, like auto or summer mode. 
And is it possible to use any temperature sensor connected to the raspberry and send its value over the bus to the Theta?

**Comment by phiresky** [comment, marked as answer]
Right now with this code is only one-way communitcation - it's not possible to send data of any kind to the theta. How I'm doing it is purely using the info from the Theta for analysis and to get data into home assistant - temperature settings I'm only doing via smart thermostats/valves further up in the heating system connected to home assistant. (which the theta indirectly reacts to because then it needs to heat more or less).

I have no idea what the protocol for changing stuff would look like and I haven't seen anyone else analyzing that part. I didn't try myself because it's probably pretty easy to break the thing that way.

**Reply by stefka141** [reply]
Thanks for your quick respond! 

For me it is not obvious how you connected the wires. Do you only connected Rx and Tx and no Gnd? Whereby Tx is optional, because you are not sending anything.

**Reply by phiresky** [reply]
only Rx and tx. tx acts like ground for rx
